Sentinel-dVPN-node/main.sh

```diff
@@ -42,26 +44,28 @@ declare -A config_mappings=(
     ["RPC_ADDRESS"]="rpc_address"
     ["SIMULATE_AND_EXECUTE"]="simulate_and_execute"
     ["PEERS"]="peers"
+	["IPV4_ADDRESS"]="ipv4_address"
+    ["REMOTE_URL"]="remote_url"
     ["INTERVAL_SET_SESSIONS"]="interval_set_sessions"
     ["INTERVAL_UPDATE_SESSIONS"]="interval_update_sessions"
     ["INTERVAL_UPDATE_STATUS"]="interval_update_status"
-    ["IPV4_ADDRESS"]="ipv4_address"
+    ["API_ADDRESS"]="listen_on"
     ["GIGABYTE_PRICES"]="gigabyte_prices"
     ["HOURLY_PRICES"]="hourly_prices"
-    ["REMOTE_URL"]="remote_url"
 )
 
 # Update each variable in config.toml.
 for var in "${!config_mappings[@]}"; do
     update_config "$var" "${config_mappings[$var]}" "$CONFIG_PATH"
 done
+sed -i.bak -e "s|^listen_port *=.*|listen_port = $LISTEN_PORT|;" "$V2RAY_CONFIG_PATH"
```

Comment:
why not using the `config_mappings`?
just add there these two:

```
["LISTEN_PORT"]="listen_port"
["TRANSPORT"]="transport"
```

and the `update_config` bits should work:

```
update_config "LISTEN_PORT" "listen_port" "$V2RAY_CONFIG_PATH"
update_config "TRANSPORT" "transport" "$V2RAY_CONFIG_PATH"
```